packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/node/NodePropertiesPanel.vue

```diff
@@ -33,35 +33,36 @@
 import NodeBadge from './NodeBadge.vue'
 
 import { DK_NODE_PROPERTIES_PANEL_OFFSET_TOP, DK_NODE_PROPERTIES_PANEL_WIDTH } from '../../constants'
-import NodeFormCall from './NodeFormCall.vue'
-import NodeFormServiceRequest from './NodeFormServiceRequest.vue'
-import NodeFormResponse from './NodeFormResponse.vue'
-import NodeFormJq from './NodeFormJq.vue'
-import NodeFormStatic from './NodeFormStatic.vue'
+import NodeFormCall from '../node-forms/NodeFormCall.vue'
+import NodeFormServiceRequest from '../node-forms/NodeFormServiceRequest.vue'
+import NodeFormResponse from '../node-forms/NodeFormResponse.vue'
+import NodeFormJq from '../node-forms/NodeFormJq.vue'
+import NodeFormStatic from '../node-forms/NodeFormStatic.vue'
 
 import { KSlideout } from '@kong/kongponents'
 import { getNodeTypeDescription } from './node'
-import type { NodeInstance } from '../../types'
 import { computed } from 'vue'
+import { useEditorStore } from '../store/store'
+
+const store = useEditorStore()!
```

Comment:
Shall we add some guard like:

```ts
if (!store) {
  throw new Error('Editor state is not provided. Ensure you are using provideEditorStore in a parent component.')
}
```

Reply:
I created a `useEditorStoreOrThrow` to replace all the references of `useEditorStore`